### ISSUE TITLE
Unable to train with google colab

### ISSUE THREAD

**Issue by JosephineTa** [AlexeyAB/darknet#7492]
Hi
I am trying to detect mask via google colab, but  it gave me this error
Can anyone help?
I am new in this

![image](https://user-images.githubusercontent.com/80425718/110695706-26208b80-8225-11eb-89f9-8a609e200915.png)

I have my darknet.py file and I could see that there is load_network there, but not sure why it still gave me such error

**Comment by BelkinaKate** [comment]
I have the same problem and struggling with this for weeks already...Plz help

**Comment by radhakrishnayadav93** [comment]
Yes, i am also facing the same issue while running on webcam images and videos.

![image](https://user-images.githubusercontent.com/66212777/181767044-7fcc1511-f7a5-4244-b0fe-e97eefd3e500.png)
